packages/storybook/src/schematics/component-story/index.ts

```diff
@@ -0,0 +1,146 @@
+import {
```

Comment:
We may want to move this to `@nrwl/angular` since it is specific to Angular.

Then we can do the following:

```bash
ng g @nrwl/angular:story
ng g @nrwl/react:story
```